### PR TITLE
Broken Link

### LINKED ISSUE #15367: Update link on article
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/developers/overview/github-developer-program

### What changes are you suggesting?

Just need to update the link ([Let us know about your integration](https://github.com/contact?form%5Bsubject%5D=New+GitHub+Integration)) as it just sends to the general support page instead of a specific form with the correct subject.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by BenjaminMichaelis**
### Why:

Closes #15367

### What's being changed:

Link doesn't work with the subject line automatically, trying to add a new link that will send user straight to a form that they can share the integration information with GitHub. If there is a better link I can switch it but this is the best one I found thus far.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@BenjaminMichaelis Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by ethanpalm** [comment]
Hi @BenjaminMichaelis, thanks for creating this PR. Could you describe a bit more or share a screenshot of the error you're encountering with the current link? I'm having trouble reproducing it, and when I navigate to the link, I get the form with a prefilled subject of "New GitHub Integration." Thank you!

**Comment by BenjaminMichaelis** [comment]
> Hi @BenjaminMichaelis, thanks for creating this PR. Could you describe a bit more or share a screenshot of the error you're encountering with the current link? I'm having trouble reproducing it, and when I navigate to the link, I get the form with a prefilled subject of "New GitHub Integration." Thank you!

Of course!
image
When clicking on this link, it looks like it is optimized to automatically populate a message to github with the subject "New GitHub Integration" as the link is https://github.com/contact?form%5Bsubject%5D=New+GitHub+Integration but this just sends you currently to https://support.github.com/request so I was trying to update the link to automatically pull up a contact form and populate the subject line, and the closest I got to this is the change I pushed which at least got the user to a contact form directly, just without the subject prefilled. This is to make it as easy as possible for the user!

**Comment by BenjaminMichaelis** [comment]
> @BenjaminMichaelis that is great context, thank you. ⚡
> 
> What browser are you using? I've tested on Chrome, Firefox, and Safari, but I am getting the autopopulated form like in the screenshot below.
> 

That is interesting.... I am using chrome. Here is a video of my result.

https://user-images.githubusercontent.com/22186029/153952087-b4749689-c3a0-4c96-a9d4-c43695a55e25.mp4

**Comment by ethanpalm** [comment]
Interesting indeed. Thanks for sharing that video. For now, I'm looping the Docs engineering team in on this issue to see what might be going on. I'll post an update here once I've got some more info.

File: content/developers/overview/github-developer-program.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Email <a href="mailto:partnerships@github.com">partnerships@github.com</a> to re
 ## Have an integration that works with GitHub?
 
 Awesome! We'd love to have you be part of the program. Here’s how you can spread the word:</p>
-* [Let us know about your integration](https://github.com/contact?form[subject]=New+GitHub+Integration)
+* [Let us know about your integration](https://support.github.com/contact?tags=rr-general-technical)
```
Comment (ethanpalm) [inline content/developers/overview/github-developer-program.md:31]:
```suggestion
* [Let us know about your integration](https://support.github.com/contact?tags=rr-general-technical&form[subject]=New+GitHub+Integration)
```

A small addition to autofill the subject line with `New GitHub Integration` and keep parity with the original link.

**Review by ethanpalm — DISMISSED** [review]
Thanks for your work on this @BenjaminMichaelis. The link you've added is great since it'll work whether someone is signed into or out of GitHub. I've suggested a small change to autofill the subject line of the form.

I haven't been able to determine why we were seeing different behavior with the old link, but your new one is definitely the better place to direct people so I don't want to hold this PR up longer while we dig into that.

Thank you for taking the time to contribute to the GitHub docs! I'll get this small change in, update the branch, and merge it 🚀

**Review by ethanpalm — APPROVED** [review]

File: content/developers/overview/github-developer-program.md (hunk 1 above)
Comment (BenjaminMichaelis) [reply content/developers/overview/github-developer-program.md]: Ahh that's the addition needed to autopopulate subject, couldn't quite figure it out when just trying to add data from the old link.

**Pull request merged**